### ISSUE TITLE
Calculate the heat loss caused by laser ventilation using sensors

### ISSUE THREAD

**Issue by goatchurchprime** [DoESLiverpool/somebody-should#1266]
In spite of the fact we're running 300W machines, the laser room is often the most uncomfortably cold in the hackspace due to having the windows open to let the air vents out.  

This is being looked at by issue #5, and I hope we can poke a temperature sensor through a sealed gap in the steel tube to get at the value.  (The rate of flow should be a constant, to be determined from the spec of the ventilation fans.)

Current plan is to let the air into the laser room through a vent into the main area.  I am concerned that this will cause lots of cold air to be sucked out of the hallway into the main space and make the rest of us miserable, and we won't know why.  

Now, there exist small-scale heat recovery units for bathrooms, like this one:
    http://www.kair.co.uk/product/kair-heat-recovery-room-ventilator/
If the heat loss calculations (based on the vent air temperature) indicate that it could work, we should find a hackable heat recovery system based on all the accessible pipage and a vent coming in from outside.  Then the laser cutters would be a source of warm toastiness, rather than making everyone cold.

**Comment by seanspotatobusiness** [comment]
Is it not just mainly because the window is open for the hoses to hang
out of or have they been switched over to the solid, permanent
ventilation system now?

**Comment by Sean-anotherone** [comment]
Good idea, should be possible to retrofit something to the duct system once we have that installed.  It will be necessary to have a laser room window open just a little to ensure sufficient flow to clear the machines, but not having one wide open will make a big difference on it's own.

 a plate heat exchanger is well within capabilities and can be made with heavy duty kitchen foil supported on chicken wire.  Might need an additional hole for the intake air though.

**Comment by mrchrisadams** [comment]
@goatchurchprime If I understand this correctly, you're proposing

- install this thing in the with the laser cutter room
- have it pipe the bad air out, but recover the heat while doing so, to recirculate elsewhere in the space

How much do these things cost? I hadn't seen them before.
